### PR TITLE
Fix test expectations about sharing custom role

### PR DESCRIPTION
**Pull request by PVince81**
## Description
TODO

## Related Issue
- Fixes https://github.com/owncloud/phoenix/issues/1897

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] actual fix

### CODE REVIEW

File: tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature [hunk 1]
```diff
@@ -388,5 +388,24 @@ Feature: Sharing files and folders with internal users
...
+      | field       | value              |
+      | uid_owner   | user2              |
+      | share_with  | user1              |
+      | file_target | /simple-folder (2) |
+      | item_type   | folder             |
+      | permissions | <permissions>      |
+    Examples:
+      | role        | displayed-role | collaborators-permissions     | displayed-permissions | permissions                         |
+      | Viewer      | Viewer         | share                         | share                 | read, share                         |
+      | Editor      | Editor         | share                         | share                 | all                                 |
```
[the remark this answers is not in the document]
Comment (PVince81) [reply tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature]: sorry, seems I messed up the tests. going to adjust

File: tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature [hunk 2]
```diff
@@ -388,5 +388,24 @@ Feature: Sharing files and folders with internal users
...
       | Editor      | Editor         | share                         | share                 | all                                 |
       | Custom Role | Custom role    | share, create                 | share, create         | read, share, create                 |
-      | Custom Role | Editor         | change, share                 | share                 | read, change, share                 |
+      | Custom Role | Custom role    | change, share                 | change, share         | read, change, share                 |
       | Custom Role | Editor         | delete, share, create, change | share                 | read, share, delete, change, create |
+
+  Scenario Outline: share a file with another internal user assigning a role and the permissions
+    Given user "user2" has logged in using the webUI
+    When the user shares file "testimage.jpg" with user "User One" as "<role>" with permissions "<collaborators-permissions>" using the webUI
+    Then user "User One" should be listed as "<displayed-role>" in the collaborators list for folder "simple-folder" on the webUI
```
[the remark this answers is not in the document]
Comment (PVince81) [reply tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature]: indeed..

**Comment by PVince81** [comment]
@LukasHirt I've adjusted the tests.

The file tests pass as we correctly map "share, create" to "Editor".

The folder tests will fail for "Custom role" which is supposed to stay custom instead of "Editor" for "share, create".

**Comment by LukasHirt** [comment]
IMO this was taken care of in #2373. Roles are now being displayed correctly for both folder and file. Not sure about the new scenario you implemented there - is that needed or can we close this pr? @PVince81

**Comment by PVince81** [comment]
closing as obsolete

**Pull request closed without merging**